types/marko/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-var-requires": false,
```

Comment:
instead of disabling lint rules in the `tslint.json`, can you disable them per line as [described here](https://github.com/DefinitelyTyped/DefinitelyTyped#lint).

Reply:
thanks, fair enough.